packages/@aws-cdk/aws-codepipeline-actions/lib/cloudformation/pipeline-actions.ts

```diff
@@ -513,6 +513,487 @@ export class CloudFormationDeleteStackAction extends CloudFormationDeployAction
+   *
+   * A list of AWS accounts or organizational unit IDs where stack set instances should be created/updated.
+   *
+   * Accounts:
+   *
+   * Example:
+   *
+   * 111111222222,333333444444
+   *
+   *
+   * OrganizationalUnitIds:
+   *
+   * Note
+   * This parameter is optional for the SERVICE_MANAGED permissions model and is not used for the SELF_MANAGED permissions model. Do not use this if
+   * you select OrganizationsAutoDeployment.
+   *
+   * The AWS organizational units in which to update associated stack instances.
+   *
+   * Example:
+   *
+   * ou-examplerootid111-exampleouid111,ou-examplerootid222-exampleouid222
+   *
+   * One of deploymentTargets or deploymentTargetsPath is required, but not both.
+   *
+   * @default No deployment target. This or deploymentTargetsPath is required.
+   */
+  readonly deploymentTargets?: string | Array<string>;
+
+  /**
+   * Note
+   * For the SERVICE_MANAGED permissions model, you can provide either accounts or organizational Unit IDs for deployment targets. For the
+   * SELF_MANAGED permissions model, you can only provide accounts.
+   *
+   * Note
+   * When this parameter is selected, you must also select Regions.
+   *
+   * A list of AWS accounts or organizational unit IDs where stack set instances should be created/updated.
+   *
+   * Accounts:
+   *
+   * The following example shows the file contents for accounts.txt `sourceArtifact.atPath("accounts.txt")`, where sourceArtifact is a
+   * CodePipeline Artifact.
+   *
+   * [
+   *     "111111222222"
+   * ]
+   *
+   *
+   *
+   * OrganizationalUnitIds:
+   *
+   * Note
+   * This parameter is optional for the SERVICE_MANAGED permissions model and is not used for the SELF_MANAGED permissions model. Do not use this if
+   * you select OrganizationsAutoDeployment.
+   *
+   * The AWS organizational units in which to update associated stack instances.
+   * You can provide organizational unit IDs as an Array, a literal list or a file path:
+   *
+   * The following example shows the file contents for accounts.txt `sourceArtifact.atPath("OU-IDs.txt")`, where sourceArtifact is a
+   * CodePipeline Artifact.
+   *
+   * [
+   *     "ou-examplerootid111-exampleouid111",
+   *     "ou-examplerootid222-exampleouid222"
+   * ]
+   *
+   * One of deploymentTargets or deploymentTargetsPath is required, but not both.
+   *
+   * @default No deployment target. This or deploymentTargets is required.
+   */
+  readonly deploymentTargetsPath?: codepipeline.ArtifactPath;
+
+  /**
+   * Note
+   * When this parameter is selected, you must also select DeploymentTargets.
+   *
+   * A list of AWS Regions where stack set instances are created or updated. Regions are updated in the order in which they are entered.
+   *
+   * Enter a list of valid AWS Regions in the format Region1,Region2, as shown in the following example.
+   *
+   * [ "us-west-2", "us-east-1" ]
+   */
+  readonly regions: string | Array<string>;
```

Comment:
1. Same comment as above (do not use union types).
2. How does this interact with `region` above?

Reply:
it doesn't - region is the CodePipeline action region, regions is the StackSets regions

Reply:
I wonder then, what's the benefit of setting `region` here?